index.html

```diff
@@ -201,8 +201,27 @@ <h3>Skills</h3>
           </ul>
         </section>
       </section>
+
+      <footer class="contact-wrapper">
+        <h2 class="contact-title">Contact me</h2>
```

Comment:
- [ ] Please make sure to stick to the design and have a white title for the footer. Currently, it is black.